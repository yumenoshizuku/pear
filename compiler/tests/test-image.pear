Main()(
window.Window(),
window.Size(1026, 1026),
window.Title("Chii"),
window.Resizable("no"),
window.BorderWidth(2),
image.Image("chii.jpg"),
window.Contain(image),
window.Showall(),
)
