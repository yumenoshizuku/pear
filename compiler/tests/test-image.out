#include <stdio.h>
#include <stdlib.h>
#include <gtk/gtk.h>
#include <glib.h>
#include <glib/gstdio.h>


int main(int argc, char** argv)
{
	GtkWidget* window;
	GtkWidget* image;
	
	gtk_init(&argc, &argv);
	window = gtk_window_new(GTK_WINDOW_TOPLEVEL);
	g_signal_connect(window, "destroy", G_CALLBACK(gtk_main_quit), NULL);
	gtk_window_set_default_size(GTK_WINDOW(window), 1026, 1026);
	gtk_window_set_title(GTK_WINDOW(window), "Chii");
	gtk_window_set_resizable(GTK_WINDOW(window), TRUE);
	gtk_container_set_border_width(GTK_CONTAINER(window), 2);
	image = gtk_image_new_from_file("chii.jpg");
	gtk_container_add(GTK_CONTAINER(window), image);
	gtk_widget_show_all(window);
	gtk_main();
	return 0;
}

